### PR TITLE
No waiting for minting, skip transfer

### LINKED ISSUE #86: Remove passphrase prompt ahead of checkout
## Overview

Passphrase prompt should not happen until after pack opening.

### LINKED ISSUE #87: Do not wait for minting and do not trigger transfer during checkout
## Overview

During checkout, do not wait for minting to complete and do not trigger transfer endpoint.

### LINKED ISSUE #88: Update UI after pack opening
## Overview

After the 3D pack opening screen, we need to add UI for the following:

- Show NFTs are being minted
- New button to trigger the transfer of NFTs
- Modal to accept passphrase and show transfer status

### PR DESCRIPTION
**Pull request by smonn**
This resolves quite a few things:

Closes #87 
Closes #86 
Closes #88 

**Old flow was:**

1. Select a pack to purchase
2. Enter passphrase
3. Enter and submit payment details
4. Mint (wait for minting to finish) and transfer pack
5. Open pack

**New flow is:**

1. Select a pack to purchase
2. Enter and submit payment details
3. Mint pack (do not wait for minting to finish)
4. Open pack
5. Enter passphrase and transfer pack

~Note: Will still need to add an extra status check before enabling the transfer button to make sure the pack has been fully minted. Also need some design input before this is considered fully done.~

### CODE REVIEW

**Review by jakerainis — APPROVED** [review]
👍

File: apps/web/src/components/pack-grid/pack-grid.tsx [hunk 1]
```diff
@@ -118,12 +119,17 @@ export default function PackGrid({
...
-      </Button>
+      {onTransfer && (
+        <Button
+          className={css.viewCollectionButton}
+          onClick={onTransfer}
+          disabled={packCards.length === 0 || !enableTransfer}
+        >
+          {t('common:actions.Save to My Collection', {
+            count: packCards.length,
+          })}
```
Comment (smonn) [inline apps/web/src/components/pack-grid/pack-grid.tsx:130]:
```suggestion
          {t('common:actions.Save to My Collection')}
```

**Review by IanPirro — APPROVED** [review]
👌

**Review by dallashuggins — APPROVED** [review]
🔥🔥

**Pull request merged**